docs/image-variants.md

```diff
@@ -0,0 +1,40 @@
+# Image variants
+
+The Flex API allows requesting custom image variants by accompanying an image
+variant sparse attribute parameter with a corresponding image variant
+definition:
+
+``` js
+sdk.listings.show({
+  id: listingId,
+  include: ["images"],
+  "fields.image": ["variants.my-variant"],
+  "imageVariant.my-variant": "w:640;h:1280;fit:scale"
```

Comment:
Is there a reason why this variant name uses kebab-case?

Reply:
The reasoning is that our [predefined image variants](https://www.sharetribe.com/api-reference/marketplace.html#image-resource-format) use kebab-case too.